### DISCUSSION TITLE
VanJS-Firebase Todo List App with Routing and Authentication

### DISCUSSION THREAD

**Discussion by kwameopareasiedu** [vanjs-org/van#215] [Show and tell]
I recently completed a full featured todo-list app with VanJS and Firebase as a backend. It features authentication, routing, styling (TailwindCSS) and uses the `vanjs-ext`. It's meant as a showcase of how VanJS can be integrated into a full-stack stack

Live Demo: https://kwameopareasiedu.github.io/vanjs-todo/
Github Repo: https://github.com/kwameopareasiedu/vanjs-todo

![image](https://github.com/vanjs-org/van/assets/26204283/0d8b3711-e38e-4cc8-ab9a-dbe0b7e38926)

**Comment by Tao-VanJS** [comment]
Thanks for sharing! I added your app into VanJS [Community Examples](https://vanjs.org/demo#community-examples).
